filter.d/sabnzbd-auth.conf

```diff
@@ -1,11 +1,20 @@
-## Version 2022/08/06
+## Version 2022/08/11
 # Fail2Ban filter configuration for sabnzbd
 
 [INCLUDES]
 before = common.conf
 
 [Definition]
 
-failregex = ^.*(Unsuccessful login attempt from|Refused connection( with hostname "<HOST>")? from:|(API Key missing|API Key incorrect|Authentication missing).* your 3rd party program:) (?:\(X-Forwarded-For: (<HOST>)(?:,[^']*)?\)|(<HOST>)) \[.*\]$
+failregex = ^.*API Key incorrect, Use the api key from Config->General in your 3rd party program: .* \(X-Forwarded-For: <HOST>\) \[[^\]*]\]$
```

Comment:
I keep thinking that using a conditional for x-forwarded-for is better, but i dont know if `( .* \(X-Forwarded-For:
)? <HOST>(\))?` would add too many steps

Reply:
I considered this, but the issue is the syntax would need `<HOST>` to potentially be in two places within one line of the regex, which I think is what was causing the issue in the first place.
`(?:\(X-Forwarded-For: (<HOST>)(?:,[^']*)?\)|(<HOST>)) \[.*\]$` contains `<HOST>` twice, which I believe might have been the problem.

Reply:
Also, the lines here
https://github.com/sabnzbd/sabnzbd/blob/f2361c49b4429c21e9f44374b27b2324753d3667/sabnzbd/interface.py#L119-L129
indicate that when `X-Forwarded-For` is present, the output will be:
```
<AN-INTERNAL-IP> (X-Forwarded-For: <HOST>) [<User-Agent>]
```
And when `X-Forwarded-For` is NOT present, the output will be:
```
<HOST> [<User-Agent>]
```

I think I misread your initial comment, and your regex might work, but it's a bit less obvious than I'd like (my original regex was worse).

the conf for `sshd` is even more complicated, but it seems to cover a lot https://github.com/linuxserver/fail2ban-confs/blob/62ed71ae23144c97ff87f5d85a24a1b8d1e11dbc/filter.d/sshd.conf#L42-L62